locales/zh_CN/php.json

```diff
@@ -135,19 +135,19 @@
     "size.file": ":Attribute 大小必须为 :size KB。",
     "size.numeric": ":Attribute 大小必须为 :size。",
     "size.string": ":Attribute 必须是 :size 个字符。",
-    "slug_input_explanation": "此 slug 用于在 URL 中识别您的团队。",
+    "slug_input_explanation": "此slug短标签用于在 URL 中识别您的团队。",
```

Comment:
```diff
- 此slug短标签用于在
+ 此 slug 短标签用于在
```

Reply:
@zeturn, can you fix that?

Reply:
@andrey-helldar maybe you can edit it directly and then merge it?